pkg/pool-stable-phantom/test/StablePhantomPool.test.ts

```diff
@@ -770,7 +774,7 @@ describe('StablePhantomPool', () => {
 
         describe('update', () => {
           const itUpdatesTheRateCache = (action: (token: Token) => Promise<ContractTransaction>) => {
-            const newRate = fp(1.5);
+            const newRate = fp(4.5);
```

Comment:
Why is this sensitive to the particular newRate value? It fails with 1.5, but succeeds with other values (such as 4.5).

Should the newRate be coordinated somehow (e.g., pulled out into a constant or something?)

Reply:
Ah forgot to mention this in the description, sorry. Essentially each token is assigned an initial rate of `1.n`, where `n` is their index. This means the 5th token was assigned a rate of 1.5, and the test failed because the new rate was equal to the old rate. This will never happen with a rate of 4.5.